infra/terraform/secure_pipelines/mob_async_backend_pl.tf

```
@@ -36,6 +36,9 @@ locals {
 
       IncludePromotion = "No"
 
+      ContainerSignerKmsKeyArn = one(data.aws_cloudformation_stack.container_signer_dev[*].outputs["ContainerSignerKmsKeyArn"])
```

I appreciate this approach follows a pattern used to get outputs from other cloudformation stacks, so it makes sense to copy it. However this approach (of using a `data` resource) is done to get outputs from _cloudformation stacks in different aws accounts_. 

In this case you're getting an output from the `devplatform-container-signer` stack from the same terraform configuration. This means you can remove that layer of indirection and reference the output directly.

Here is the resource: https://github.com/govuk-one-login/mobile-id-check-async/blob/484ed22aa788691bd68686a6202844a941190975/infra/terraform/base_stacks/artifact.tf#L35C1-L49C2

And here is how you would reference it: 

```suggestion
      ContainerSignerKmsKeyArn = aws_cloudformation_stack.container_signer_dev[0].outputs["ContainerSignerKmsKeyArn"])
```

Now that resource is only made for the `dev` and `build` accounts. But since you're only using the output when in the `dev` and `build` accounts you shouldn't need to use any additional logic here.